www/~/%username/emails/verify.html.spt

```diff
@@ -4,14 +4,17 @@ from datetime import timedelta
 
 from aspen import Response
 from aspen.utils import utcnow
-from gratipay.utils import emails, get_participant
+from gratipay.utils import emails, get_participant, b64decode_s
 
 [-----------------------------------------------------------------------------]
 
 participant = get_participant(state, restrict=False)
 banner = '~' + participant.username
 if participant == user.participant:
-    email = request.qs.get('email', '')
+    if 'email' in request.qs:
+        email = request.qs['email']
+    else:
+        email = b64decode_s(request.qs.get('email64', ''), default='')
```

Comment:
> Probably better to re-ticket this removal and do this after a while when we are confident we are not losing any users because of this.

Commented in b3422ccd14ebd726f573ff22b329f4bfa0671951.